src/go/go.mod

```diff
@@ -1,4 +1,4 @@
-module apiview
+module apiviewgo
 
 go 1.14
```

Comment:
Should this get bumped to 1.16 to be consistent with the rest of our packages?

Reply:
Given this is an internal tool I don't think it really matters.  Of course, bumping it won't hurt either.

Reply:
Sure. I can push that change along with readme change.